pcsx2/CDVD/InputIsoFile.cpp

```diff
@@ -22,15 +22,21 @@
 
 static const char* nameFromType(int type)
 {
-    switch(type)
-    {
-        case ISOTYPE_CD:		return "CD";
-        case ISOTYPE_DVD:		return "DVD";
-        case ISOTYPE_AUDIO:		return "Audio CD";
-        case ISOTYPE_DVDDL:		return "DVD9 (dual-layer)";
-        case ISOTYPE_ILLEGAL:	return "Illegal media";
-        default:				return "Unknown or corrupt";
-    }
+	switch (type)
+	{
+		case ISOTYPE_CD:
+			return "CD";
+		case ISOTYPE_DVD:
+			return "DVD";
+		case ISOTYPE_AUDIO:
+			return "Audio CD";
+		case ISOTYPE_DVDDL:
+			return "DVD9 (dual-layer)";
+		case ISOTYPE_ILLEGAL:
+			return "Illegal media";
+		default:
+			return "Unknown or corrupt";
+	}
```

Comment:
This is my personal preference for switch/case
```cpp
switch (type)
{
case ISOTYPE_CD:
	return "CD";
case ISOTYPE_DVD:
	return "DVD";
case ISOTYPE_AUDIO:
	return "Audio CD";
case ISOTYPE_DVDDL:
	return "DVD9 (dual-layer)";
case ISOTYPE_ILLEGAL:
	return "Illegal media";
default:
	return "Unknown or corrupt";
}
```

Reply:
I personally prefer the cases to be tabbed in, it just looks weird in the github changes cos it's tabs

Reply:
> I personally prefer the cases to be tabbed in

Same.

Reply:
Well I think I've contributed most of the current style at this point so I'll concede.

Reply:
As for doing the entire case on one line, I mean, eh, it's situational I think... The problem is VS especially likes to fight against that styling

Reply:
I'm personally just not a fan of one-liners.

stuff like
```cpp
if (something) return nullptr;
```

Really make it hard for me to tell what I'm even looking at and if I need to modify this code it's extra annoying that I have to break it into multiple lines and add brackets. I did add a bit in the style guide about being able to do this:
```cpp
if (something)
   return nullptr;
```

Mostly because that's what most seem to do in the project but it's personally not my preference as I find it inconsistent and if I want to add a printf or something to that I have to add the brackets myself. I'm alright with extending that exception to cases, though.

I'm not sure I understand why peeps want to save space in the vertical direction. I'd rather save it horizontal.

Reply:
I agree, I prefer things to be underneath rather than all squashed on to the same line to save space

Reply:
> I'm not sure I understand why peeps want to save space in the vertical direction. I'd rather save it horizontal.

Personally for the switch cases, it's less about saving space and more about the fact that in switch statements with the form `case SOMETHING: return Something;`, it keeps all the cases together and return values together like a table which makes the parts that differ stand out, making it much easier for me to read.  Especially since in C these often have common prefixes, not alternating between cases and return values makes it really easy to skip the prefix and start reading from the actual name

![image](https://user-images.githubusercontent.com/3315070/93841772-44d09d00-fc5a-11ea-9e94-3347b2684caf.png)